extract_V1_firmware.py

```diff
@@ -5,108 +5,106 @@
 import os
 import sys
 
-def extract(filename, dirname):
-	print("Extract file", filename)
-	print("to directory", dirname)
-
-	print("Press enter to continue or CTRL-C to abort")
-	input()
 
-	os.makedirs(dirname, exist_ok=True)
-
-	with open(filename, "rb") as fd:
-		content = fd.read()
+def extract(filename, dirname):
```

Comment:
Hey, you removed all those pretty tabs! ;)

[Yes, I use python with tabs if nobody is looking at me :) ]

Reply:
That was ruff-format 😏